spec/activity_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
```

Comment:
that you can drop from here as `rails_helper` loads `spec_helper`

https://github.com/kalabiyau/NonSolus/blob/master/spec/rails_helper.rb#L2